amundsen_application/static/images/favicons/dev/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Same question here to understand what this file achieves.

Reply:
This one is the manifest file on the line of Progressive Web Apps.
It sets up the homescreen icon and start screen for when users add amundsen to their homescreen in their phones (also in windows desktops)

Reply:
I see, also from the generator?

Reply:
Yep!
This one is more useful though, it works also for windows desktop machines.